### ISSUE TITLE
Incorrect parsing of host portion

### ISSUE THREAD

**Issue by cage2** [fukamachi/quri#38]
Hi @fukamachi !

I sincerely hope that you are fine and you recovered from the problems you mentioned elsewhere. 

Anyway i think i have found a problem with parsing that is perhaps related to #31 

``` lisp
(let ((uri (quri:uri "//example.host/a/path/")))
           (quri:uri-host uri)) ; => nil
```
i think the value of this expression should be `example.com`.

Thank you again for all your work!
Bye!
C.

**Comment by Ambrevar** [comment]
Fixed with #39.  
@cage2 Thanks for reporting!
@jathd Thanks for fixing it!

**Comment by cage2** [comment]
On Tue, Feb 16, 2021 at 03:15:08AM -0800, Pierre Neidhardt wrote:
> Fixed with #39.
> @cage2 Thanks for reporting!
> @jathd Thanks for fixing it!

Thanks to you all for your works! :)

Bye!
C.
